### PR TITLE
feat(rules): Add `simple-import-sort` plugin and rule

### PR DESCRIPTION
**Pull request by billyvg**
This will auto-sort our imports

### CODE REVIEW

File: packages/eslint-config-sentry-app/index.js [hunk 1]
```diff
@@ -120,35 +120,45 @@ module.exports = {
...
-            position: 'after',
-          },
+        groups: [
+          // Side effect imports.
+          ['^\\u0000'],
+
+          // Node.js builtins. You could also generate this regex if you use a `.js` config.
+          // For example: `^(${require("module").builtinModules.join("|")})(/|$)`
+          [
+            '^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)',
```
Comment (EvanPurkhiser) [inline packages/eslint-config-sentry-app/index.js:138]: Any reason we're not using the thing in the comment?

Comment (billyvg) [reply packages/eslint-config-sentry-app/index.js:138]: Ah, was just copy pasting, we could use it since we assume everyone's on the same version of node

Comment (billyvg) [inline packages/eslint-config-sentry-app/index.js:138]:
```suggestion
          // Node.js builtins. 
          // For example: ``
          [
            '^(${require("module").builtinModules.join("|")})(/|$)',
```

File: packages/eslint-config-sentry-app/index.js [hunk 2]
```diff
@@ -120,35 +120,45 @@ module.exports = {
...
-            pattern: 'admin*/**',
-            group: 'internal',
-            position: 'after',
-          },
+        groups: [
+          // Side effect imports.
+          ['^\\u0000'],
+
+          // Node.js builtins. 
+          // For example: ``
```
Comment (EvanPurkhiser) [inline packages/eslint-config-sentry-app/index.js:136]: 🤔

**Review by EvanPurkhiser — APPROVED** [review]

**Pull request merged**